client/app/bundles/course/assessment/submission/components/CommentCard.jsx

```diff
@@ -30,7 +30,13 @@ const styles = {
     display: 'flex',
     alignItems: 'center',
     justifyContent: 'space-between',
-    backgroundColor: grey100,
+    backgroundColor: lightBlue100,
```

Comment:
Normal comments can probably leave as grey100 so that users have continuity of UI experience.
Only the new feature has a new differentiating colour.

Reply:
Noted!